pkg/vm/vm.go

```diff
@@ -236,12 +236,84 @@ func (v *VM) Step() {
 	}
 }
 
+// StepInto behaves the same as “step over” in case if the line does not contain a function it otherwise
+// the debugger will enter the called function and continue line-by-line debugging there.
+func (v *VM) StepInto() {
+	ctx := v.Context()
+
+	if ctx != nil {
```

Comment:
Really?

Reply:
you mean that if ctx == nil we won't reach stepInto ever? or what?=)

Reply:
There is something wrong here, either you should set some other flag, or the condition should be inverted. At the moment this branch will always win (because you do normally have some context) and then lead to `v.HasStopped()` returning true also.

Reply:
With this implementation of `StepInto` this test fails for me
https://github.com/neo-project/neo-vm/blob/master/tests/neo-vm.Tests/Tests/OpCodes/Arrays/NEWARRAY.json#L7

Reply:
@roman-khimov heh you were right.. It should be `ctx == nil `